v03_pipeline/lib/reference_data/config.py

```diff
@@ -533,4 +536,14 @@ def custom_mpc_select(ht):
             'custom_import': import_locus_intervals,
         },
     },
+    'local_constraint_mito': {
```

Comment:
I've left `high_constraint_region_mito` and can remove in a follow up so that we can have a clean handover with a change of the naming.

Reply:
Ticket here: https://github.com/broadinstitute/seqr-loading-pipelines/issues/846